src/common.jl

```diff
@@ -13,10 +13,11 @@ using
 using
     Statistics,
     Bootstrap
-
+using Rasters
 import GeoDataFrames as GDF
 import GeoFormatTypes as GFT
 import ArchGDAL as AG
+import GeoInterface as GI
```

Comment:
Maintain spacing please

```suggestion

using Rasters
import GeoDataFrames as GDF
import GeoFormatTypes as GFT
import ArchGDAL as AG
import GeoInterface as GI
```